src/AirtableHttpClient.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yoanbernabeu\AirtableClientBundle;
+
+use Symfony\Component\HttpClient\ScopingHttpClient;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class AirtableHttpClient extends ScopingHttpClient
```

Comment:
This is the concrete client used for Http calls, it extends `ScopingHttpClient` to have a default configuration (`id` `apiKey` `base_uri` and `accept header`